safe/gui/tools/minimum_needs/needs_profile.py

```diff
@@ -334,3 +331,20 @@ def remove_profile(self, profile):
             os.path.join(
                 str(self.root_directory), 'minimum_needs', profile + '.json')
         )
+
+    def load_old_profile(self, locale_minimum_needs_dir):
```

Comment:
Can you rename it to 'move_old_profile'.
The idea is not to load it, but to move the folder.